src/index.js

```diff
@@ -258,14 +351,30 @@ export default class ToggleBlock {
    */
   _iterateOnItems(items, toggleIndex) {
     let index = toggleIndex;
-    if (this.data.status === 'closed') {
-      for (let i = 0; i < items; i += 1) {
-        this.api.blocks.getBlockByIndex(index += 1).holder.setAttribute('hidden', true);
-      }
-    } else {
-      for (let i = 0; i < items; i += 1) {
-        this.api.blocks.getBlockByIndex(index += 1).holder.removeAttribute('hidden');
-      }
+
+    switch (this.data.status) {
+      case 'closed':
+        if (items > 0) {
+          for (let i = 0; i < items; i += 1) {
+            this.api.blocks.getBlockByIndex(index += 1).holder.setAttribute('hidden', true);
```

Comment:
@juliansg022 avoid chaining too many function calls. My suggestion is to use object destructing also is good for readability.

Reply:
@juanhurtado10 You're right, I'll change those instructions.